ind.ie.Gnomit.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>ind.ie.Gnomit.desktop</id>
+  <name>Gnomit</name>
+  <developer_name>Aral Balkan (Ind.ie)</developer_name>
+  <update_contact>hello@ind.ie</update_contact>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-or-later</project_license>
+  <summary>Git commit message editor</summary>
+	<description>
+	  <p>Gnomit is a simple Git commit message editor for Gnome, inspired by the excellent Komet app on macOS by Mayur Pawashe.</p>
+	  <p>Features:</p>
+	  <ul>
+	    <li>Highlights overflow of subject line when it exceeds 69 characters.</li>
+      <li>Inserts empty line between subject line and the rest of the message.</li>
```

Comment:
indenting!

Reply:
Fixed in https://source.ind.ie/gnome/gnomit/gjs/commit/bcabb3d532c82348c2bc8b9522374d131d0bfdc4 – will include it in the next push to this PR once I handle some of the other issues also.